wordpress-mobile/WordPress-iOS
# Update No Results View image displayed on connectivity issue

Fixes #11031 
This PR depends on #11030 
We want to show the same illustration when the full screen (no results view) is displayed in case of connectivity issue.

![img_3294](https://user-images.githubusercontent.com/1335657/53135315-64648880-3540-11e9-8e1b-8c9d4b7ad0c6.PNG)
![img_3295](https://user-images.githubusercontent.com/1335657/53135316-64648880-3540-11e9-980c-4a28170022bf.PNG)

To test:
1. Run the app.
2. Go offline
3. Go over any of the full pages and see that the illustration is consistent in all pages.
4. Check that the full screen illustration is the same in case the screen appears due to other reason (will update testing steps)
    4.1. Create a new site
    4.2. Go to Media.
    4.3. See that the illustration and the copy are media specific.
    4.4. Go to site pages.
    4.5. See that the illustration and the copy are site pages specific.
    4.6. Go to Drafts tab.
    4.7. See that the copy is draft pages specific.
    4.8. Go to blog posts
    4.9. Delete the About post.
    4.10. See that the illustration and the copy are Blog posts specific.
    etc'
Update release notes:
"Updated No Results View illustration and copy displayed on connectivity issue."
